### ISSUE TITLE
Issue on docs

### ISSUE THREAD

**Issue by Arsik36** [FlatFilers/Guides#545]
Path: /guides/dynamic-configurations

I am not sure how to propose a PR to fix this so creating an issue. We have 4 code snippets here, a pair of how to create a workbook dynamically for `js` and `ts`, and same goes for duplicate sheet. Is there a way we can tab those, similar to how we have those tabbed here, for readability? https://flatfile.com/docs/core-paths/part4#joining-fields

**Comment by hansjhoffman** [comment]
@Arsik36 It looks like all the code samples are now tabbed on [dynamic configurations](https://flatfile.com/docs/guides/dynamic-configurations) as you were suggesting. Could you please confirm this is what you were asking for?

**Comment by Arsik36** [comment]
Yep, from everyhting I see this is no longer an issue, closing :)
